### DISCUSSION TITLE
Upgrade Milvus v2.3.10 to v2.4

### DISCUSSION THREAD

**Discussion by ducanh997** [milvus-io/milvus#34896] [Q&A and General discussion]
I'm currently using distributed Milvus v2.3.10 in production and would like to upgrade to v2.4. Can I achieve this simply by changing the tag in my Helm configurations, or is there a more involved upgrade process I should follow?

Thanks!

**Comment by yhmo** [comment]
Follow the doc to upgrade v2.3 to v2.4. No special process if your version is greater than v2.2.3.
https://milvus.io/docs/upgrade_milvus_cluster-helm.md
